src/chrome/content/rules/GoNewSoft.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="GoNewSoft.com">
+        <target host="gonewsoft.com" />
+        <target host="www.gonewsoft.com" />
+		
+		<rule from="^http://(?:www\.)?gonewsoft\.com/" to="https://gonewsoft.com/"/>
```

Comment:
I would like to be removed www

for example :
http://GoNewSoft.com
http://www.GoNewSoft.com
https://www.GoNewSoft.com

Redirect to :
https://GoNewSoft.com